galata/test/jupyterlab/sidebars.test.ts

```diff
@@ -205,3 +211,74 @@ test.describe('Sidebars', () => {
     expect(tableOfContentsElementRole).toEqual('region');
   });
 });
+
+const elementAriaLabels = {
+  'jp-running-sessions': [
+    'Open Tabs Section',
+    'Kernels Section',
+    'Language servers Section',
+    'Terminals Section'
```

Comment:
I guess we will need these to reflect new additions:

```suggestion
    'Language servers Section',
    'Recently Closed Section',
    'Workspaces Section',
    'Terminals Section'
```